docs/c-runtime-library/reference/findnext-functions.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "_findnext, _findnext32, _findnext32i64, _findnext64, _findnext64i32, _findnexti64, _wfindnext, _wfindnext32, _wfindnext32i64, _wfindnext64, _wfindnext64i32, _wfindnexti64"
 ms.date: "11/04/2016"
-api_name: ["_wfindnext", "_findnext"]
+api_name: ["_findnext, _findnext32, _findnext32i64, _findnext64, _findnext64i32, _findnexti64, _wfindnext, _wfindnext32, _wfindnext32i64, _wfindnext64, _wfindnext64i32, _wfindnexti64"]
```

Comment:
These api_name entries either need to be wrapped in double quotes individually, or none of them should have quotes.

Reply:
Editing...